src/main/resources/static/codegen/built-cas3-api-spec.yml

```diff
@@ -327,6 +327,7 @@ components:
           $ref: '#/components/schemas/Task'
         users:
           type: array
+          description: Users to which this task can be allocated
```

Comment:
I wouldn't normally nick pick like this but as this commit is all about polishing language then I think "Users to _whom_ this task can be allocated" may be a bit clearer

Reply:
Yes, agreed. Fixed :)